txpipe/twopoint_fourier.py

```diff
@@ -732,9 +731,20 @@ def compute_noise_analytic(self, i, j, k, maps, f_sky, workspace):
             ndens = (
                 metadata["tracers/lens_density"][i] * 3600 * 180 / np.pi * 180 / np.pi
             )
+            with self.open_input("mask", wrapper=True) as f:
+                info = f.read_map_info("mask")
```

Comment:
Info variable is not needed here.

Also we could also pass the map in as a variable to avoid reading it again for every bin pair.

Reply:
Yes, I removed info implemented your suggestion of passing it as a variable.